evm_loader/program/src/payment.rs

```diff
@@ -94,12 +94,16 @@ pub fn transfer_from_operator_to_deposit<'a>(
     debug_print!("deposit_sol_info {:?}", deposit_sol_info);
 
     let rent_via_sysvar = Rent::get()?;
-    if rent_via_sysvar.lamports_per_byte_year * deposit_sol_info.data.borrow().len() as u64 * 2_u64 > deposit_sol_info.lamports() {
+    let rent_exempt_balance = rent_via_sysvar.minimum_balance(deposit_sol_info.data.borrow().len());
```

Comment:
`deposit_sol_info.data.borrow().len()` can be simplified to `deposit_sol_info.data_len()`